addons/mail/tests/test_mail_thread.py

```
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests import tagged
+
+
+@tagged('mail_thread', 'mail')
+class TestMailThread(MailCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_search_message_partner_ids(cls):
+        '''
+            The purpose of this test case is to check that _search_message_partner_ids does
+            not raise a traceback when called on a false domain, instead, raise an access error
```

Says it should raise an access error but that doesn't seem to be what we want